reference/orders.v2.oas2.yml

```diff
@@ -3626,7 +3626,8 @@ components:
           title: Tracking Carrier
           description: |-
             Tracking carrier for the shipment.
-            Acceptable values include an empty string (`""`) or one of the valid tracking-carrier values that you can view and download as a .CSV file [here](https://github.com/bigcommerce/dev-docs/blob/master/assets/csv/tracking_carrier_values.csv).
+            Acceptable values include an empty string (`""`) or one of the valid [tracking-carrier values](https://github.com/bigcommerce/dev-docs/blob/master/assets/csv/tracking_carrier_values.csv) and 
+            the [downloadable .CSV file](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/csv/tracking_carrier_values.csv).
```

Comment:
same as comment at link 696, plus here you have an artificially divided line. use whatever you decide on for line 696, then make sure it's on one line.

Reply:
Relevant slack conversation [here](https://bigcommerce.slack.com/archives/C039JSWHH50/p1670865268099679?thread_ts=1670848230.004619&cid=C039JSWHH50)